opentech/apply/categories/blocks.py

```diff
@@ -34,6 +34,7 @@ class CategoryQuestionBlock(FormFieldBlock):
     )
     category = ModelChooserBlock('categories.Category')
     multi = BooleanBlock(label='Multi select', required=False)
+    required = BooleanBlock(label='Required', required=False)
```

Comment:
Good spot, would prefer change is base class.

https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/stream_forms/blocks.py#L47

Reply:
sorted. 704ada7